### DISCUSSION TITLE
Redundant harvesting without removing plots from the original plotting system

### DISCUSSION THREAD

**Discussion by mumtazali2** [Chia-Network/chia-blockchain#4346] [Q&A]
You should be able to harvest same plots from many systems instead of removing them from one and adding them to a new harvester?  My plots are on the main system where they were created, I want to have them harvested on a new harvester for redundancy and efficiency purpose.

**Comment by LarHope** [comment]
Or better yet. in relatively large harvester network, we should ENABLE  Load balance in the harvester protocol. So few farmer in the same LAN could work with different harvesters in the same loop. It will help the fault tolerance when one of farmers fail.

**Comment by SackofFluor** [comment]
So your Idea is, that technically you could win multiple times with just one block?
That doesn't make sense in any conceivable way...

**Reply by pluto2021** [reply]
Probably only one will get registered on the blockchain, as it is a same plot, under a same Wallet. If found a match, that reward of only 2XCH should go to the wallet associated. Since only one match is found, only 2XCH should be distributed, otherwise, the whole scheme would not work well.

**Reply by mumtazali2** [reply]
Yes, there should be only one reward. How it can be done technically, it is upto the developers, but idea is I should be able to make plots available from another system for performance reasons because on my main system I am doing plotting which makes it slow to respond to blockchain.
